integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/ReceiptPolling.java

```diff
@@ -74,9 +76,10 @@ public void dpiReceiptsScheduledTask() {
         if(externalReceipt != EMPTY_KVITTERING) {
             Audit.info("Got receipt (DPI)", externalReceipt.logMarkers());
             final String id = externalReceipt.getId();
-            MessageReceipt receipt = messageReceiptRepository.findOne(id);
-            receipt = externalReceipt.update(receipt);
-            messageReceiptRepository.save(receipt);
+            Conversation conversation = conversationRepository.findOne(id);
+            // TODO: add receipt update to conversation
+//            receipt = externalReceipt.update(receipt);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)